plugins/scaffolder-backend-module-gitlab/api-report.md

```diff
@@ -201,17 +201,63 @@ export const createPublishGitlabMergeRequestAction: (options: {
   JsonObject
 >;
 
+// @public
+export const editGitlabIssueAction: (options: {
+  integrations: ScmIntegrationRegistry;
+}) => TemplateAction<
+  {
+    repoUrl: string;
+    projectId: number;
+    issueIid: number;
```

Comment:
Is it really `Issue IID`? or should this be `issueId`?

Reply:
EDIT: Nevermind I see that it's `response.iid` further down.

Reply:
Yeah, Gitlab has 'Issue ID, and Issue IID' 
So like it could be isuse 100000, but it's IID is 5 of a specific project that its related to.
The whole plugin bases itself off the passed in Group and Project via a `repoUrl`